doc/input-output-reference/src/overview/group-radiative-convective-units.tex

```diff
@@ -1011,7 +1011,7 @@ \subsubsection{Inputs}\label{inputs-5-022}
 
 \paragraph{Field: Hydronic Tubing Length}\label{field-hydronic-tubing-length}
 
-This field is the total length of pipe embedded in the surface named above in the surface name field. The length of the tube should be entered in meters and is used to determine the effectiveness of heat transfer from the fluid being circulated through the tubes and the tube/surface. Longer tubing lengths result in more heat will be transferred to/from the radiant surface to the circulating fluid. Note that if the user elects to autosize this field that a standard zone thermostat such as would be used for a forced air system must be defined as autosizing calculations are based on the zone thermostat value and not on the radiant system control values.
+This field is the total length of pipe embedded in the surface named above in the surface name field. The length of the tube should be entered in meters and is used to determine the effectiveness of heat transfer from the fluid being circulated through the tubes and the tube/surface. Longer tubing lengths result in more heat will be transferred to/from the radiant surface to the circulating fluid. Note that if the user elects to autosize this field that a standard zone thermostat such as would be used for a forced air system must be defined as autosizing calculations are based on the zone thermostat value and not on the radiant system control values. In addition, when the user opts to autosize this calculation, the tube spacing from the construction of each surface associated with this system is used along with each individual surface area to make an approximation of the tubing length.  
```

Comment:
@rraustad Ok, thanks for catching that.  I have fixed this so hopefully it is grammatically and technically correct now.

@Myoldmopar @mjwitte  This should be ready to merge, but I am a little concerned that my GitHub desktop app did something weird again.  I updated from develop but then I got an error message about a file that I have changed (doc file) before I did the merge.  That seemed to confuse the app.  I committed the change and then did another update from develop and it showed more files that needed to be updated. I don't think there were any conflicts so hopefully it isn't going to be a problem.  If there is an issue, let me know.